specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json

```diff
@@ -0,0 +1,331 @@
+          "items": {
+            "type": "number",
+            "format": "float"
+          }
+        },
+        "lowerMargins": {
+          "type": "array",
+          "description": "LowerMargins contain lower margin of each input point. LowerMargin is used to calculate lowerBoundary, which equals to expectedValue - (100 - sensitivity)*lowerMargin. Points between the boundary can be marked as normal ones in client side. The index of the array is consistent with the input series.",
+          "items": {
+            "type": "number",
+            "format": "float"
+          }
+        },
+        "isAnomaly": {
+          "type": "array",
+          "description": "IsAnomaly contains anomaly properties for each input point. True means an anomaly either negative or positive has been detected. The index of the array is consistent with the input series.",
+          "items": {
+            "type": "boolean"
+          }
+        },
+        "isNegativeAnomaly": {
+          "type": "array",
+          "description": "IsNegativeAnomaly contains anomaly status in negative direction for each input point. True means a negative anomaly has been detected. A negative anomaly means the point is detected as an anomaly and its real value is smaller than the expected one. The index of the array is consistent with the input series.",
+          "items": {
+            "type": "boolean"
+          }
+        },
+        "isPositiveAnomaly": {
+          "type": "array",
+          "description": "IsPositiveAnomaly contain anomaly status in positive direction for each input point. True means a positive anomaly has been detected. A positive anomaly means the point is detected as an anomaly and its real value is larger than the expected one. The index of the array is consistent with the input series.",
+          "items": {
+            "type": "boolean"
+          }
+        }
+      }
+    },
+    "LastDetectResponse": {
+      "type": "object",
+      "properties": {
+        "period": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Frequency extracted from the series, zero means no recurrent pattern has been found."
+        },
+        "suggestedWindow": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Suggested input series points needed for detecting the latest point."
+        },
+        "expectedValue": {
+          "type": "number",
+          "format": "float",
+          "description": "Expected value of the latest point."
+        },
+        "upperMargin": {
+          "type": "number",
+          "format": "float",
```

Comment:
Is this really a float? Please verify that you are using the correct number type (here and in other places)

Reply:
It should be an integer